Cargo.toml

```diff
@@ -0,0 +1,3 @@
+[workspace]
+members=["backend/rust-bindings"]
+resolver="2"
```

Comment:
Could you please not use workspaces where there is not a critical need? Espeically a workspace with one member is just overhead.

Reply:
When opening the project from root the rust analyser starts. Is there an overhead having it?

Reply:
There is a mental overhead. Now there is yet another configuration file that controls behaviour.

I am not sure what you mean by the rust-analyzer. At least in emacs you can add arbitrary folders to the workspace so this is not neeed.